composer.json

```diff
@@ -30,7 +30,7 @@
     "require-dev" : {
         "kriswallsmith/buzz" : "~0.8",
         "guzzle/http"        : "~3",
-        "guzzlehttp/guzzle"  : "~5.2"
+        "guzzlehttp/guzzle": "^6.1"
```

Comment:
it should allow both actually (and we should adapt the Travis setup to run tests against both)